src/main/java/seedu/address/model/schedule/Event.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.model.schedule;
+
+import java.time.Duration;
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+/**
+ * Represents a scheduled Event.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Event {
+
+    private final EventDescription eventDescription;
+    private final LocalDate date;
+    private final LocalTime time;
+    private final Duration duration;
```

Comment:
Is there any benefit to using the Duration library as compared to just int or double? Is there more compatibility to LocalTime using Duration?

Reply:
Yup, you can refer to this https://www.baeldung.com/java-period-duration for more information about Duration. It has a method that can calculate the duration between 2 LocalTime instances. Alternatively, we can also create our own Duration class, but I don't think that is very necessary.